src/applications/disability-benefits/996/components/IntroductionPage.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+
+import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import CallToActionWidget from '../../../../platform/site-wide/cta-widget';
+import { toggleLoginModal } from '../../../../platform/site-wide/user-nav/actions';
+import { focusElement } from '../../../../platform/utilities/ui';
```

Comment:
Feel free to just make all these paths relative to `platform`; we have module aliasing available to us.

Reply:
Done.